validation-test/stdlib/Slice.swift.gyb

```diff
@@ -69,6 +91,12 @@ SliceTests.test("${Slice}/init(base:bounds:)") {
   }
 }
 
+SliceTests.test("${Slice}/baseProperty") {
+  let referenceCollection = ReferenceCollection()
+  let testSlice = ${Collection}(base: ReferenceCollection(), bounds: 0..<1)
```

Comment:
Oh yeah this isn't going to work at all. It looks like I didn't stage all my changes (sign I was working too late 😐). To start with `${Collection}` isn't the right variable, and it's not using `referenceCollection` at all in this iteration.  I was intending to just use Slice, like you suggested, but I also figured out how to test a few more by filtering out RangeReplaceable and Mutable slices.